app/models/page.rb

```diff
@@ -23,6 +26,25 @@ def feature_flag_name
     "page_#{slug}"
   end
 
+  # def has_a_landing_page?
+  #   Page.find_by(landing_page: true)
+  # end
+
+  # def landing_page_path
+  #   return unless has_a_landing_page?
+
+  #   landing_page = has_a_landing_page?
+  #   landing_page.path
+  # end
+
+  def self.landing_page
+    find_by(landing_page: true)
+  end
+
+  def self.landing_page_exists?
```

Comment:
This seems to never be used, can we remove it?

Reply:
I actually changed some of the code within `/pages/_form.html.erb` today, so this method is now used!